internal/service/alarms/internal/alarms_server.go

```diff
@@ -0,0 +1,90 @@
+package internal
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/google/uuid"
+	api "github.com/openshift-kni/oran-o2ims/internal/service/alarms/api/generated"
+)
+
+type AlarmsServer struct {
+	//TODO: Add anything that's needed during request e.g DB conn
+}
+
+// AlarmsServer implements StrictServerInterface. This ensures that we've conformed to the `ServerInterface` with a compile-time check
+var _ api.StrictServerInterface = (*AlarmsServer)(nil)
+
+func (a AlarmsServer) GetSubscriptions(ctx context.Context, request api.GetSubscriptionsRequestObject) (api.GetSubscriptionsResponseObject, error) {
+	// TODO implement me
+	panic("implement me")
+}
+
+func (a AlarmsServer) CreateSubscription(ctx context.Context, request api.CreateSubscriptionRequestObject) (api.CreateSubscriptionResponseObject, error) {
+	// TODO implement me
+	panic("implement me")
+}
+
+func (a AlarmsServer) DeleteSubscription(ctx context.Context, request api.DeleteSubscriptionRequestObject) (api.DeleteSubscriptionResponseObject, error) {
+	// TODO implement me
+	panic("implement me")
+}
+
+func (a AlarmsServer) GetSubscription(ctx context.Context, request api.GetSubscriptionRequestObject) (api.GetSubscriptionResponseObject, error) {
+	// TODO implement me
+	panic("implement me")
+}
+
+func (a AlarmsServer) GetAlarms(ctx context.Context, request api.GetAlarmsRequestObject) (api.GetAlarmsResponseObject, error) {
+	// TODO implement me
+
+	// Fill out more details
+	p := api.ProblemDetails{
+		Detail: "invalid `filter` parameter syntax",
+		Status: http.StatusBadRequest,
+	}
+	return api.GetAlarms400ApplicationProblemPlusJSONResponse(p), nil
+}
+
+func (a AlarmsServer) GetAlarm(ctx context.Context, request api.GetAlarmRequestObject) (api.GetAlarmResponseObject, error) {
+	// TODO implement me
+	alrm := api.AlarmEventRecord{
```

Comment:
could we spell out `alarm` rather than `alrm`

Reply:
updated